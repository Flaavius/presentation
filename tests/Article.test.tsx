import React from "react";
import { shallow } from "enzyme";

import { Article } from "../src/Components/Article";

describe("Article", () => {
  const article = shallow(<Article title="Hello World" id={2} />);
  it("Renders Article Component", () => {
    expect(article).toMatchSnapshot();
  });
});
