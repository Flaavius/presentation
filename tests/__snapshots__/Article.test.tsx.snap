// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Renders Article Component 1`] = `
<div
  className="article"
  onClick={[Function]}
>
  <div
    className="thumb"
  >
    <Component
      id={2}
      noShadow={true}
      size={
        Object {
          "height": 150,
          "width": 150,
        }
      }
    />
  </div>
  <div
    className="text"
  >
    <p>
      Hello World
    </p>
  </div>
</div>
`;
